string name
uint32 num_of_messages
float64 bytes_per_second
float32 avg_msgs_per_sec
float32 avg_bytes_per_msg
uint64 total_bytes